src/McpResponse.ts

```diff
@@ -295,6 +305,27 @@ export class McpResponse implements Response {
               ),
             };
           }
+          if (item instanceof AggregatedIssue) {
+            const count = item.getAggregatedIssuesCount();
+            const filename = item.getDescription()?.file;
+            const rawMarkdown = filename ? getIssueDescription(filename) : null;
+            if (!rawMarkdown) {
```

Comment:
if there is not markdown title, let's skip the item because it is not very useful for the clients (and log a debug error)